### DISCUSSION TITLE
Welcome to SeaweedFS Discussions!

### DISCUSSION THREAD

**Discussion by chrislusf** [seaweedfs/seaweedfs#1733] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by PlatonaBoy** [comment]
Currently, I am selecting a Distributed File System, and I feel that seaweedfs is good. Is there any case?

**Reply by chrislusf** [reply]
yes. Please check the wiki first.

**Reply by PlatonaBoy** [reply]
我们正在做技术选型，发现seaweedfs功能确实很好，但相对于其他很多分分布式文件存储系统，他们都有很多公司级别的案例，我想知道我们的seaweedfs目前有哪些公司在使用

**Reply by chrislusf** [reply]
Please check the wiki.
